persistence.js

```diff
@@ -393,12 +393,12 @@ LevelPersistence.prototype.incomingGetPacket = function (client, packet, cb) {
 }
 
 LevelPersistence.prototype.incomingDelPacket = function (client, packet, cb) {
-  var key = INCOMING + client.id + ':' + packet.messageId
+  const key = INCOMING + client.id + ':' + packet.messageId
```

Comment:
We should really use `${INCOMING}${clientId}:${package.messageId}` imo

Reply:
Go for the template syntax yes!